stylesheet/main.css

```diff
@@ -1,56 +1,106 @@
 @CHARSET "UTF-8";
-body{
+
+body {
     background-color: #FFE7B8;
     color: #FFBF3E;
     /* missing font-family */
 }
-    
 
-    
+
+
 a {
-  color : #8259AF;
-  text-decoration: none;
+    color: #8259AF;
+    text-decoration: none;
 }
+
 header nav a {
-    color : #5E5369;
-    }
+    color: #5E5369;
+}
 
 header ul {
-  list-style-type: none;
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
 }
 
 main ul {
-  list-style-type: circle;
+    list-style-type: circle;
 }
-    
-   
+
+
 
 
 p {
-  color: orange;
+    color: orange;
 }
 
 .bio p {
-  text-indent: 3em;
+    text-indent: 3em;
 }
 
 em {
-  color: green;
+    color: green;
 }
 
-.byline{
+.byline {
     font-size: 70%;
-    color: red; 
-    color: rgba(255,255,255,0.5)
+    color: red;
+    color: rgba(255, 255, 255, 0.5)
 }
+
 #page-copyright {
-    color:#8259AF;
+    color: #8259AF;
     text-transform: uppercase;
     letter-spacing: 3px;
 }
 
-footer{
+footer {
     font-size: 12px;
     text-align: center;
 }
 
+.width-wrap {
+    background-color: #92a8d1;
+    max-width: 1024px;
+    padding-right: 2em;
+    padding-left: 2em;
+    margin-bottom: 3em;
+    margin-left: auto;
+    margin-right: auto;
+
+    border-left: 1px solid rgba(0,0,0,0.3);
+    border-right: 1px solid rgba(0,0,0,0.3);
+    border-bottom: 1px solid rgba(0,0,0,0.3);
+
+    box-shadow: 0 0 4px rgba(0,0,0,0.2);
+}
+
+.mugshot img {
+    box-shadow: 2px 2px 3px rgba(0,0,0,0.5);
+}
+

```

Comment:
This rule makes the box shadow on your mugshot image.  Not the descendant selector that picks only images in the .mugshot element